src/components/ProgressBar/ProgressBar.module.scss

```diff
@@ -0,0 +1,91 @@
+@import "~styles/variables.scss";
+
+$progress-bar-height: $base-spacing * 0.75;
+
+.progressBarWrapper {
+  display: inline-flex;
+  align-items: center;
+  width: 100%;
+  min-width: $base-spacing * 12;
+}
+
+.progressBar {
+  overflow: hidden;
+  background-color: $brand-grey;
+  height: $progress-bar-height;
+  border-radius: $progress-bar-height / 2;
+  flex: 1;
+}
+
+// non-visible element is the actual progress percentage but allows the fill element to animate on load
+// this will also animate if the percentage changed after component mount
+.progressBarFillWidth {
+  height: 100%;
+  transition: width 0.2s ease-out;
+}
+
+.progressBarFill {
+  height: 100%;
+  width: 100%;
+  background-color: currentColor;
+  border-radius: $progress-bar-height / 2;
+  -webkit-animation: progress-bar-in 0.4s ease-out;
+  animation: progress-bar-in 0.4s ease-out;
+}
+
+.percentage {
+  margin-left: $base-spacing;
+  color: currentColor;
+}
+
+@-webkit-keyframes progress-bar-in {
```

Comment:
are these needed? The webkit addons? Isn't this included automatically via webpack?

Reply:
I did a quick project search when I came to the animation for @keyframes to see any standard and I saw one that was defined explicitly like this. I'll remove it though, I noticed autoprefixer is included.